src/components/record-modal/record-modal.css

```diff
@@ -0,0 +1,183 @@
+@import "../../css/colors.css";
+@import "../../css/units.css";
+
+.modal-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 1000;
+    background-color: rgba(94, 102, 125, .95);
+}
```

Comment:
Yeah we no longer have a generic small modal component (the large modal component is now full-screen and uses a text filter in the nav bar, so isn't general enough anymore). Should probably be refactored.